Readme.md

```diff
@@ -34,6 +36,18 @@ If you have implemented a SaltyRTC client, task or server :+1: and you would lik
 
 * [saltyrtc-server-python](https://github.com/saltyrtc/saltyrtc-server-python)
 
+## Releases
+
+When we release a new version of the protocol, a (new) task or the
+chunking specification, a tag will be added in the following format:
+`<protocol|chunking>-<version>` or for tasks
+`task-<task-name>-<version>`.
+
+Note that the specifications' versions are independent from each other.
+In case a new version of a specification breaks backwards compatibility
+to another specification, it will include a section stating how
+compatibility is affected.
```

Comment:
I'd suggest adding a `Changelog` section into the specification once a specification is being updated. Having one log for several specs seems confusing to me. What do you think?